### PR TITLE
Add KubeStateMetrics and PrometheusNodeExporter charts to OKS

### PR DESCRIPTION
**Pull request by jaronoff97**
To be merged after #1264

### CODE REVIEW

File: charts/opentelemetry-kube-stack/values.yaml [hunk 1]
```diff
@@ -891,3 +891,175 @@ opAMPBridge:
...
+# Prometheus Configuration #
+# (optional)               #
+############################
+# This configuration sections allows for a direct replacement of the kube-prometheus-stack
+# chart where the collector scrapes the same metrics as the default prometheus installation.
+
+## Component scraping kube state metrics
+##
+kubeStateMetrics:
+  enabled: false
```
Comment (TylerHelmuth) [inline charts/opentelemetry-kube-stack/values.yaml:905]: Are we able to move `enabled` under `kube-state-metrics` or do they have a json schema that wont allow it?

File: charts/opentelemetry-kube-stack/values.yaml [hunk 2]
```diff
@@ -891,3 +891,175 @@ opAMPBridge:
...
+      #   targetLabel: nodename
+      #   replacement: $1
+      #   action: replace
+
+  selfMonitor:
+    enabled: false
+
+## Deploy node exporter as a daemonset to all nodes
+##
+nodeExporter:
```
Comment (TylerHelmuth) [inline charts/opentelemetry-kube-stack/values.yaml:978]: What is the node exporter? Can you add more details in the comments

File: charts/opentelemetry-kube-stack/values.yaml [hunk 3]
```diff
@@ -891,3 +891,175 @@ opAMPBridge:
...
+# This configuration sections allows for a direct replacement of the kube-prometheus-stack
+# chart where the collector scrapes the same metrics as the default prometheus installation.
+
+## Component scraping kube state metrics
+##
+kubeStateMetrics:
+  enabled: false
+
+## Configuration for kube-state-metrics subchart
+##
```
Comment (TylerHelmuth) [inline charts/opentelemetry-kube-stack/values.yaml:908]: Lets link to the kube state metrics chart.

File: charts/opentelemetry-kube-stack/values.yaml (hunk 2 above)
Comment (TylerHelmuth) [reply charts/opentelemetry-kube-stack/values.yaml:978]: If this is for the `prometheus-node-exporter`, can the configs be combined?

**Review by TylerHelmuth — COMMENTED** [review]
Lets link in the comments to any subchart's readmes.

File: charts/opentelemetry-kube-stack/values.yaml (hunk 1 above)
Comment (jaronoff97) [reply charts/opentelemetry-kube-stack/values.yaml:905]: that may work, this is mostly for kube-prom-stack compatibility which matches this.

File: charts/opentelemetry-kube-stack/values.yaml (hunk 2 above)
Comment (jaronoff97) [reply charts/opentelemetry-kube-stack/values.yaml:978]: same as above

File: charts/opentelemetry-kube-stack/values.yaml (hunk 1 above)
Comment (TylerHelmuth) [reply charts/opentelemetry-kube-stack/values.yaml]: Like you could copy from your values.yaml into this one?

Comment (jaronoff97) [reply charts/opentelemetry-kube-stack/values.yaml]: [yes](https://github.com/prometheus-community/helm-charts/blob/main/charts/kube-prometheus-stack/values.yaml#L2024-L2027), i think there's some value in doing that for compatibility, but I think this is overall cleaner...

Comment (TylerHelmuth) [reply charts/opentelemetry-kube-stack/values.yaml]: Ok ya I'm fine with keeping it 100% compatible.

**Review by TylerHelmuth — APPROVED** [review]

**Pull request merged**